src/plugins/explore/opensearch_dashboards.json

```diff
@@ -20,7 +20,7 @@
     "expressions",
     "dashboard"
   ],
-  "requiredBundles": ["opensearchDashboardsReact", "opensearchDashboardsUtils", "dashboard"],
+  "requiredBundles": ["opensearchDashboardsReact", "opensearchDashboardsUtils", "dashboard", "contextProvider"],
```

Comment:
why is this required? What is a requiredBundle btw?

It seems weird that contextProvider is a required something but is itself an optional plugin

Reply:
so requiredBundle is build time dependency and requiredPlugin is run time dependency. For optionalPlugins, it is for if we can use some plugin's service, but not needed.

I think requiredBundles is the right one because we are using it as a utility library (importing hooks/components).. so we need contextProvider's TypeScript definitions to compiler, ather than as a service provider (using plugin services)